torch/nested/_internal/nested_tensor.py

```diff
@@ -260,10 +260,16 @@ def backward(ctx, gO: torch.Tensor):  # type: ignore[override]
 # Not actually a view!
 class ViewNestedFromBuffer(torch.autograd.Function):
     @staticmethod
-    def forward(ctx, values: torch.Tensor, offsets: torch.Tensor):  # type: ignore[override]
+    def forward(
+        ctx,
+        values: torch.Tensor,
+        offsets: torch.Tensor,
+        metadata_cache: Optional[Dict[str, Any]] = None,
```

Comment:
Since you added a new non-differentiable input, you'll need to return another "None" argument in your backward as well